CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,120 @@
+# Code of Conduct
+
+We follow the [CNCF Code of Conduct][cncf-coc].
+
+As part of our pledge to respect all people, in both live in-person and online
+interactions, we are committed to providing a friendly, safe and welcoming
+environment for all, regardless of gender, gender identity and expression,
+sexual orientation, disability, physical appearance, body size, race, religion,
+native language, operating system choice, current software stack or prior
+experience.
+
+In keeping with this commitment, we offer the following guidelines:
+
+* Welcome newcomers at any stage of expertise.
+  * Everyone has something to contribute.
+  * Everyone deserves access to materials and community that will help them
+       learn.
+  * As long as an individual can be respectful and not disruptive to other
+       participants, they deserve to participate.
+* Provide open and free material.
+* Be kind and courteous.
+  * Interpret the arguments of others in good faith, offering private
+       constructive feedback when communication style bears improvement.
+  * Leave space for quieter voices.
+* Consider who is not in the room.
+  * Invite participation from experts or user community representatives
+       outside of the working group.
+  * Participate in online forums to be inclusive of those who cannot attend
+       meetings.
+* Work performed within this group, either finalized or in draft, is to be
+     used in accordance with the group [Mission and
+     Charter][charter],
+     the open source license, and to be used for the equal benefit of all
+     members of the community.
+
+## Incident handling and escalation
```

Comment:
Again, this is a dup of the CNCF CoC. There is a reporting/enforcement process in place and I don't see why it cannot be used to handle the situation here.